package.json

```diff
@@ -7,8 +7,8 @@
   "license": "MPL-2.0",
   "scripts": {
     "postinstall": "cd stumptown && npm install && git checkout package-lock.json",
-    "prettier-check": "prettier --check **/*.{js,jsx,scss,css,html}",
-    "prettier-format": "prettier --write **/*.{js,jsx,scss,css,html}",
+    "prettier-check": "prettier --check **/*.{js,jsx,json,scss,css,html}",
+    "prettier-format": "prettier --write **/*.{js,jsx,json,scss,css,html}",
```

Comment:
```suggestion
    "prettier-format": "prettier --write **/*.{js,json,scss,html}",
```

Reply:
Hang on, what's the point of this? Why are we specifying these things twice? Both here and in the .prettierignore whitelist. Can this line here be simplified so that the .prettierignore is the only thing that configures stuff?

Reply:
Hmm, on the other note, what's the reason of ignoring everything in the first line of this `.prettierignore`?

https://github.com/mdn/stumptown-renderer/blob/41ea08eef8cd0dcc2bf47d1d7c7c4beb1d0d47d1/.prettierignore#L1-L2

I feel like the intended usage of prettier is that we specify the files that we want to format in the package.json (passing it as a glob pattern), and then ignore some specific folder in `.prettierignore`

Changing it to "match everything" on the package.json, and then "ignore everything except these few patterns" is doable but feels a bit wrong 😅 

So since this is a matter of personal preference, what approach do you prefer to take?

Thanks

Reply:
I'm not sure what's right what I do know is that:

1. It's weird to specify black and white lists in the same file. 
2. It's weird that we list the extensions in two places.
3. The file is called .prettierIGNORE and by white listing extensions, are we breaking what's natural?
4. If we use `package.json` as our "white list" and the .prettierignore as our black list, do we lose any power?